### DISCUSSION TITLE
Suggestion: Rename the VS Code Language Server Extension

### DISCUSSION THREAD

**Discussion by brunoborges** [apache/netbeans#4226] [Ideas]
Hey all,

Currently your extension is published in the VS Code Marketplace under the name "Language Server".

I'd suggest you rename it to "Java Language Server by Apache NetBeans".

This will increase discoverability of your extension in the marketplace, especially when people search for the keyword "java".

**Comment by mbien** [comment]
could you bring this to the dev mailing list?

**Reply by brunoborges** [reply]
Sure, but if things should not be discussed through this GitHub repo, I suggest you disable the **Discussions** feature/tab.

**Reply by mbien** [reply]
It seems to be fairly popular, I don't see a reason to disable it. But feel free to bring that up too if you want.

**Reply by brunoborges** [reply]
Then not sure why I also need to post this suggestion to the dev list :-D

But anyhow... done.

**Reply by mbien** [reply]
because project decisions happen on the dev list for apache projects. My dev instincts tingled and told me that this isn't going to be a long discussion, its more of a request. So it makes sense to bring this directly to the dev list. Github here is more user facing.

**Comment by geertjanw** [comment]
https://github.com/apache/netbeans/pull/4295

Gj

**Comment by geertjanw** [comment]
https://github.com/apache/netbeans/pull/4295
